### PR TITLE
[css-color] Scale down values in oklab/oklch and clamp L > 100% for lab/lch/oklab/oklch

### LINKED ISSUE #39000: [css-color-4] Adjust tests for L > 100% in lab/lch functions to match spec
The tests for computed values of `lab()`, `lch()`, `oklab()`, `oklch()` where L > 100% in [css/css-color/parsing/color-computed-lab.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-color/parsing/color-computed-lab.html) need to be adjusted to the latest spec, which states

> The first argument specifies the CIE Lightness. This is a number between 0% (representing black) and 100% (representing white), Values less than 0% must be clamped to 0% at computed-value time; values greater than 100% are clamped to 100%. [[Candidate Recommendation Draft](https://www.w3.org/TR/css-color-4/#specifying-lab-lch)]

Further clarified in the Editor's Draft: 

> The first argument specifies the CIE Lightness, L. This is a number between 0% or 0 (representing black) and 100% or 100 (representing white), Values less than 0% or 0 must be clamped to 0% at parsed-value time; values greater than 100% or 100 are clamped to 100% at parsed-value time.

* https://github.com/web-platform-tests/wpt/blob/597c96814cb435d6f50f87193d368fc1595bc303/css/css-color/parsing/color-computed-lab.html#L40
* https://github.com/web-platform-tests/wpt/blob/597c96814cb435d6f50f87193d368fc1595bc303/css/css-color/parsing/color-computed-lab.html#L73

If this is as straightforward as changing the expected computed value in these two instances, I can submit a PR.

### PR DESCRIPTION
**Pull request by danburzo**
Fixes #39000. 

The PR includes the following changes:

- `oklab()`: divide `l`, `a`, and `b` by 100
- `oklch()`: divide `l`, `c` by 100
- `lab()` / `lch()` / `oklab()` / `oklch()`: clamp `l` to `[0-100]`.   

Changing these across several hundred tests has been quite hypnotic, so any extra pairs of eyes are greatly appreciated!

### CODE REVIEW

File: css/css-color/parsing/color-computed-lab.html [hunk 1]
```diff
@@ -51,29 +51,29 @@
...
-test_computed_value("color", "oklab(-40 0 0)", "oklab(0 0 0)");
-test_computed_value("color", "oklab(50 -20 0)", "oklab(50 -20 0)");
-test_computed_value("color", "oklab(50 0 -20)", "oklab(50 0 -20)");
-test_computed_value("color", "oklab(calc(50 * 3) calc(0.5 - 1) calc(1.5) / calc(-0.5 + 1))", "oklab(150 -0.5 1.5 / 0.5)");
-test_computed_value("color", "oklab(calc(-50 * 3) calc(0.5 + 1) calc(-1.5) / calc(-0.5 * 2))", "oklab(0 1.5 -1.5 / 0)");
+test_computed_value("color", "oklab(-0.4 0 0)", "oklab(0 0 0)");
+test_computed_value("color", "oklab(0.5 -0.2 0)", "oklab(0.5 -0.2 0)");
+test_computed_value("color", "oklab(0.5 0 -0.2)", "oklab(0.5 0 -0.2)");
+test_computed_value("color", "oklab(calc(0.5 * 3) calc(0.5 - 1) calc(1.5) / calc(-0.5 + 1))", "oklab(1 -0.5 1.5 / 0.5)");
+test_computed_value("color", "oklab(calc(-0.5 * 3) calc(0.5 + 1) calc(-1.5) / calc(-0.5 * 2))", "oklab(0 1.5 -1.5 / 0)");
```
Comment (svgeesus) [inline css/css-color/parsing/color-computed-lab.html:66]: lightness should be 0 not 1.5, right? (-0.5 * 3) is -1.5 which clamps to 0

**Review by svgeesus — CHANGES_REQUESTED** [review]
Spotted one error otherwise looks good!

File: css/css-color/parsing/color-computed-lab.html (hunk 1 above)
Comment (svgeesus) [reply css/css-color/parsing/color-computed-lab.html:66]: Never mind, I can't read apparently

**Review by svgeesus — APPROVED** [review]

**Comment by danburzo** [comment]
Awesome, thanks for taking the time to review this, @svgeesus!

**Pull request merged**